sys/include/suit/v4/suit.h

```diff
@@ -34,17 +34,27 @@
 extern "C" {
 #endif
 
+/**
+ * @brief   Buffer size used for Cose
+ */
 #ifndef SUIT_COSE_BUF_SIZE
-#define SUIT_COSE_BUF_SIZE      512
+#define SUIT_COSE_BUF_SIZE              (512U)
 #endif
 
-#define SUIT_V4_COMPONENT_MAX 1
+/**
+ * @brief   Maximum components used for SUIT v4
+ */
+#define SUIT_V4_COMPONENT_MAX           (1U)
 
 /**
  * @brief Supported SUIT manifest version
  */
-#define SUIT_MANIFEST_VERSION        4
-#define SUIT_VERSION        1
+#define SUIT_MANIFEST_VERSION           (4)
+
+/**
+ * @brief Current SUIT version
```

Comment:
Maybe reword this to `Current SUIT serialization format version` and add a reference to https://tools.ietf.org/html/draft-moran-suit-manifest-04#section-8.2 here